src/api/app/models/project.rb

```diff
@@ -963,12 +971,34 @@ def branch_to_repositories_from(project, pkg_to_enable, opts = {})
 
     return unless pkg_to_enable.is_channel?
 
-    # explizit call for a channel package, so create the repos for it
+    # explicit call for a channel package, so create the repos for it
     pkg_to_enable.channels.each do |channel|
       channel.add_channel_repos_to_project(pkg_to_enable)
     end
   end
 
+  def branch_remote_repositories(project_name, pkg_to_enable)
+    # handle remote package case
+    possible_ancestors = RemoteProject.new(name: project_name).possible_ancestor_names
+    possible_ancestors = [project_name] if possible_ancestors.size == 0
+
+    interconnect_project = RemoteProject.find_by(name: possible_ancestors)
+
+    return false unless interconnect_project
```

Comment:
As I explained you in the office, some test cases passes strings as a Project and before it jumped in the else case (https://github.com/openSUSE/open-build-service/pull/2932/files#diff-d80ef30560c59a982e9ca5ff125427fbL183) so nothing happened. Since remote projects are also treated as strings, it's not possible to detect whether this is a remote project or just a test case that passes a string to prevent the functionality from branching. I would need to adapt those test cases. I will have a look how much effort it is.